src/model.cpp

```diff
@@ -265,8 +268,6 @@ Status Model::reloadVersions(std::shared_ptr<model_versions_t> versionsToReload,
         }
         if (modelVersion->getStatus().getState() == ModelVersionState::END || modelVersion->getModelConfig().getBasePath() != config.getBasePath()) {
             downloadModels(fs, config, versionsToReload);
-        } else {
-            config.setLocalPath(modelVersion->getModelConfig().getLocalPath());
```

Comment:
Moving around this part may break cloud storage functionalities. I suppose unit tests pass, but it would be good to check whether regression tests pass before merge.

Reply:
localPath property is overwritten in` downloadModels()` function, so I suppose everything should work just fine